checkov/common/checks_infra/registry.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, parser: BaseGraphCheckParser = BaseGraphCheckParser(), checks
         super().__init__(parser)
         self.checks = []
         self.parser = parser
-        self.checks_dir = checks_dir if checks_dir else str(Path(__file__).parent.parent / "checks" / "graph_checks")
+        self.checks_dir = checks_dir if checks_dir else str(Path(__file__).parent.parent.parent / "terraform" / "checks" / "graph_checks")
```

Comment:
@tronxd If this is in common - maybe it should demand receiving the path to the default checks dir in the c'tor?